SWLOR.Game.Server/Service/PropertyService/StructureType.cs

```diff
@@ -1272,10 +1272,280 @@ public enum StructureType
+            "_mdrn_pl_coffful",
+            "",
+            true,
+            1)]
+        CoffeeMaker = 227,
+        [Structure("Microwave (Black)",
+            "_mdrn_pl_microwb",
+            "",
+            true,
+            1)]
+        MicrowaveBlack = 228,
+        [Structure("Bar (Wooden)",
+            "_mdrn_pl_barwood",
+            "",
+            true,
+            1)]
+        BarWooden = 229,
+        [Structure("Couch Leather - Blue",
+			"frn_couch_swlr10",
+            "",
+            true,
+            1)]
+        CouchLeatherBlue = 230,
+		[Structure("Bench, Leather Cushion (Brown)",
+			"frn_couch_swlr12",
+			"",
+			true,
+			1)]
+		BenchLeatherCushionBrown = 231,
+        //232 open
+		[Structure("Table, Plastic (Large)",
+			"_mdrn_pl_tabplal",
+			"",
+			true,
+			1)]
+		TablePlasticLarge = 233,
+		[Structure("Table, Round, Oak",
+            "_mdrn_pl_table1o",
+            "",
+            true,
+            1)]
+        TableRoundOak = 234,
+        [Structure("Metal Wall, Wide, Light",
+            "_mdrn_pl_hwall24",
+            "",
+            true,
+            1)]
+        MetalWallWideLight = 235,
+        [Structure("Store Counter (Stained",
+            "_mdrn_pl_store02",
+            "",
+            true,
+            1)]
+        StoreCounterStained = 236,
+        [Structure("Chair, Large (Grey/Red",
+            "_mdrn_pl_chair30",
+            "",
+            true,
+            1)]
+        ChairLargeGreyRed = 237,
+        [Structure("Bunkbed (Metal), Grey",
+            "_mdrn_pl_shipbnk",
+            "",
+            true,
+            1)]
+        BunkbedMetalGrey= 238,
+        [Structure("Shower, White",
+            "_mdrn_pl_shower",
+            "",
+            true,
+            1)]
+        ShowerWhite = 239,
+        [Structure("Shower, Floor Basin",
+            "_mdrn_pl_showerf",
+            "",
+            true,
+            1)]
+        ShowerFloorBasin = 240,
+        [Structure("Couch - Black and Grey",
+            "frn_couch_swlr05",
+            "",
+            true,
+            1)]
+        CouchLeatherPanelsGrey = 241,
+        [Structure("Shelves, Warehouse, Full",
+            "_mdrn_pl_shelf14",
+            "",
+            true,
+            1)]
+        ShelvesWarehouseFull = 242,
+        [Structure("Bookshelf, Jedi",
+            "swlor_0001",
+            "",
+            true,
+            1)]
+        BookshelfJedi = 243,
+        [Structure("Wooden Wall, Panels, Dark (Small)",
+            "_mdrn_pl_wwall2t",
+            "",
+            true,
+            1)]
+        MetalWallSinglePipes = 244,
+        [Structure("Metal Wall, Single, Pipes",
+            "_mdrn_pl_wwall6t",
+            "",
+            true,
+            1)]
+        WoodenWallPlanksSmall = 245,
+        [Structure("Television, Big Screen",
+            "_mdrn_pl_tvbgscr",
+            "",
+            true,
+            1)]
+        TelevisionBigScreen = 245,
```

Comment:
Duplicate 245 here which throws off the rest of the numbers

Reply:
note to self: remenant from replacing wooden wall planks (small) with metal wall. reason: metal walls made more sense then wooden. wooden wall recipe fully cleared and deleted from all 3 files.